verify.go

```diff
@@ -276,6 +279,32 @@ func (v *IDTokenVerifier) Verify(ctx context.Context, rawIDToken string) (*IDTok
 		}
 	}
 
+	// If nbf claim is provided in token, ensure that it is indeed in the past.
+	if !v.config.SkipNbfCheck {
```

Comment:
@ericchiang is this worth a seperate check or should we fold this into SkipExpiryCheck?

Reply:
Yeah, just group it under SkipExpiryCheck. No need to add another public flag.

Reply:
Done - simpler for sure, though the tests got a little more verbose as I now had to include an `exp` in all of the `nbf` test claims or they'd fail.